tools/s2i-dpdk/Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM quay.io/centos/centos:8
+FROM quay.io/centos/centos:stream8
 
 LABEL maintainer="Sebastian Scheinkman <sebassch@gmail.com>"
 LABEL io.openshift.s2i.scripts-url="image:///usr/libexec/s2i"
 LABEL io.s2i.scripts-url="image:///usr/libexec/s2i"
 
 ENV BUILDER_VERSION 0.1
-ENV DPDK_VER 20.11.1
-ENV DPDK_DIR /usr/src/dpdk-stable-${DPDK_VER}
+ENV DPDK_VER 21.11
```

Comment:
please can we use the same version as we ship in our product?

dpdk-20.11-3.el8.x86_64